src/c++/perf_analyzer/infer_context.cc

```diff
@@ -67,8 +67,8 @@ InferContext::SendSequenceInferRequest(uint32_t seq_stat_index, bool delayed)
   // Need lock to protect the order of dispatch across worker threads.
   // This also helps in reporting the realistic latencies.
   std::lock_guard<std::mutex> guard(sequence_stat_[seq_stat_index]->mtx_);
-  if (!early_exit && !sequence_stat_[seq_stat_index]->paused_) {
```

Comment:
You confirmed that the concept of paused can go away?

Reply:
Correct 👍